packages/xo-web/src/common/home-filters.js

```diff
@@ -5,15 +5,12 @@ const common = {
 export const VM = {
   ...common,
   homeFilterPendingVms: 'current_operations:"" ',
-  homeFilterNonRunningVms: '!power_state:running ',
   homeFilterHvmGuests: 'virtualizationMode:hvm ',
-  homeFilterRunningVms: 'power_state:running ',
   homeFilterTags: 'tags:',
```

Comment:
Shouldn't we remove this in another PR now that there is a proper tag selector? :thinking: 

/ping @pdonias

Reply:
Yes, it would make sense.

Reply:
I will do it.